### PR TITLE
Fix typo and duplicate config doc entries

### PR DESCRIPTION
**Pull request by danielorf**
### Motivation

Fix a typo and duplicate entries in Configuration documentation.

### Modifications

Fix a typo and duplicate entries in Configuration documentation.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): **no**
  - The public API: **no**
  - The schema: **no**
  - The default values of configurations: **no**
  - The wire protocol: **no**
  - The rest endpoints: **no**
  - The admin cli options: **no**
  - Anything that affects deployment: **no**

### Documentation

  - Does this pull request introduce a new feature?  **no**
  - If yes, how is the feature documented? **N/A**
  - If a feature is not applicable for documentation, explain why? **This change simply fixes a typo and duplicate entries in Configuration documentation.**
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation **N/A**

### CODE REVIEW

File: site2/docs/reference-configuration.md [hunk 1]
```diff
@@ -708,12 +708,6 @@ The [Pulsar proxy](concepts-architecture-overview.md#pulsar-proxy) can be config
 | brokerWebServiceURLTLS | The TLS Web service URL pointing to the broker cluster | |
 | functionWorkerWebServiceURL | The Web service URL pointing to the function worker cluster. It is only configured when you setup function workers in a separate cluster. | |
 | functionWorkerWebServiceURLTLS | The TLS Web service URL pointing to the function worker cluster. It is only configured when you setup function workers in a separate cluster. | |
-|brokerServiceURL|If service discovery is disabled, this url should point to the discovery service provider.|N/A|
-|brokerServiceURLTLS|If service discovery is disabled, this url should point to the discovery service provider.|N/A|
-|brokerWebServiceURL|This settings are unnecessary if `zookeeperServers` is specified.|N/A|
-|brokerWebServiceURLTLS|This settings are unnecessary if `zookeeperServers` is specified.|N/A|
-|functionWorkerWebServiceURL|If function workers are setup in a separate cluster, configure the this setting to point to the function workers cluster.|N/A|
```
Comment (danielorf) [inline site2/docs/reference-configuration.md:715]: There are dups in the same section

File: site2/docs/reference-configuration.md [hunk 2]
```diff
@@ -708,12 +708,6 @@ The [Pulsar proxy](concepts-architecture-overview.md#pulsar-proxy) can be config
...
-|functionWorkerWebServiceURLTLS|If function workers are setup in a separate cluster, configure the this setting to point to the function workers cluster.|N/A|
```
Comment (danielorf) [inline site2/docs/reference-configuration.md:716]: There is a dup in the same section

**Review by Huanli-Meng — APPROVED** [review]
Good catch. LGTM.

**Review by sijie — APPROVED** [review]

File: site2/docs/reference-configuration.md (hunk 1 above)
Comment (Anonymitaet) [reply site2/docs/reference-configuration.md:715]: The descriptions are not the same and should be added to the 1st property?
If so, pls check all similar cases.

image

**Review by Anonymitaet — CHANGES_REQUESTED** [review]

**Pull request merged**